### ISSUE TITLE
Support for spring boot 2.4.x

### ISSUE THREAD

**Issue by xyloman** [MarcialRosales/reactor-rabbitmq-streams-getting-started#4]
Attempting to upgrade to spring boot 2.4.x is resulting in the following exception:

```
An attempt was made to call a method that does not exist. The attempt was made from the following location:

    com.pivotal.rabbitmq.RabbitEndpoint$HttpClientSupplier.get(RabbitEndpoint.java:163)

The following method did not exist:

    'reactor.netty.http.client.HttpClient reactor.netty.http.client.HttpClient.addressSupplier(java.util.function.Supplier)'

The method's class, reactor.netty.http.client.HttpClient, is available from the following locations:

    jar:file:/.m2/repository/io/projectreactor/netty/reactor-netty-http/1.0.1/reactor-netty-http-1.0.1.jar!/reactor/netty/http/client/HttpClient.class

The class hierarchy was loaded from the following locations:

    reactor.netty.http.client.HttpClient: file:/.m2/repository/io/projectreactor/netty/reactor-netty-http/1.0.1/reactor-netty-http-1.0.1.jar
    reactor.netty.transport.ClientTransport: file:/.m2/repository/io/projectreactor/netty/reactor-netty-core/1.0.1/reactor-netty-core-1.0.1.jar
    reactor.netty.transport.Transport: file:/.m2/repository/io/projectreactor/netty/reactor-netty-core/1.0.1/reactor-netty-core-1.0.1.jar


Action:

Correct the classpath of your application so that it contains a single, compatible version of reactor.netty.http.client.HttpClient
```

**Comment by MarcialRosales** [comment]
@xyloman While working on issue #3 I have already upgraded the core dependencies to their latest: reactor, reactive rabbitmq, reactor netty, rabbitmq hop. 
I will also update the getting started guides to use 2.4.x of sprint boot. I will make sure this makes it before COB 26th Feb.

**Comment by xyloman** [comment]
Thanks for the update.

**Comment by MarcialRosales** [comment]
@xyloman I have released 0.0.8 which fixes this issue. Please take into account the following before you take this release.

IMPORTANT: This release, 0.0.8. depends on a reactor-rabbitmq-1.5.2-SNAPSHOT.jar. The reactor team is releasing 1.5.2 sometime in March (probably, March 12th). I did not want to hold you up until then therefore in order to take 0.0.8 you need to include an additional repository. You can find here how to include that second repository.
